52n-wps-algorithm/src/main/java/org/n52/wps/algorithm/annotation/AnnotatedAlgorithmIntrospector.java

```diff
@@ -180,7 +180,11 @@ private void parseClass() {
                 }
             }
         }
-        
+
+        if(this.executeMethodBinding == null) {
+            throw new RuntimeException("No execute method binding for class " + getClass().getCanonicalName());
```

Comment:
I think this reports the name of the interpreter and not the subject class (`algorithmClass`)